frontend/src/components/WofButton.vue

```diff
@@ -10,7 +10,7 @@ export default {
   name: "WofButton",
 
   props: {
-    type: {
+    variant: {
```

Comment:
Heh, my bad, I wasn't aware the type is already an attribute of the HTML button tag